src/pages/Login/Login.js

```diff
@@ -1,29 +1,45 @@
 import React, { useState } from 'react';
 import './Login.scss';
 import { useNavigate } from 'react-router-dom';
-
 const Login = () => {
   const navigate = useNavigate();
   const [inputValue, setInputValue] = useState({
-    id: '',
-    pw: '',
+    email: '',
+    password: '',
   });
-  const { id, pw } = inputValue;
-
+  const { email, password } = inputValue;
   const handleInput = e => {
     const { name, value } = e.target;
-    setInputValue(prev => ({ ...prev, [name]: value }));
+    setInputValue({ ...inputValue, [name]: value });
```

Comment:
```suggestion
    setInputValue(prev => ({ ...prev, [name]: value }));
```
이전에 작업하셨던 것 처럼 하는 방법이 더 좋아 보입니다!